docs/user/DockerUsage.md

```diff
@@ -0,0 +1,107 @@
+# Using Docker Container
+
+## Prereqs
+
+- Docker [installed on your workstation](../../README.md#docker).
```

Comment:
I don't see the value is sending customers back to the main readme at this point. Bouncing through documents is hard on the end user. In this case we previously linked them directly to docker info.

Reply:
updated to point to the dependencies file